ConservationInternational/resilienceatlas
# RA2-67: Ensuring that localisation over API works

Making sure that localisation over API works (by using `locale` param) -- there is new test which should ensure this ;)

## Testing instructions

Swagger documentation can be used for this now. Create some record which is translated (for example journey) and store two different texts for different languages. When sending request to API, set up `locale` param and verify that returned text corresponds to selected language.

## Tracking

https://vizzuality.atlassian.net/browse/RA2-67
